apps/api/src/app/translations/e2e/v1/get-translation-group.e2e-ee.ts

```diff
@@ -1,7 +1,7 @@
 import { UserSession } from '@novu/testing';
 import { expect } from 'chai';
 
-describe('get translation group - /translations/groups/:identifier (GET) #novu-v2', async () => {
+describe('get translation group - /translations/groups/:identifier (GET) #novu-v1', async () => {
```

Comment:
Is it possible to keep those tests somehow running? The problem is that we stopped running v1 tests and this caused some issues and regression without us knowing about them in translations

Reply:
I can leave them unchanged as #novu-v2